src/app/main.cpp

```diff
@@ -78,7 +78,7 @@ void sigNormalHandler(int signum);
 void sigAbnormalHandler(int signum);
 #endif
 // sys_signame[] is only defined in BSD
-const char *sysSigName[] = {
+const char *const sysSigName[] = {
```

Comment:
It's unrelated change. I was just annoyed by the warning and I fixed it. If you insist, I will remove it from this commit.

Reply:
just curious what warning?

Reply:
no previous extern declaration for non-static variable 'sysSigName'